test/users/signout.test.js

```diff
@@ -0,0 +1,36 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import app from '../../server';
+import Sessions from '../../server/controllers/Sessions';
+
+chai.use(chaiHttp);
+const { destroy } = Sessions;
+
+describe('Sign out Test', () => {
+  const sessionToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c';
+  it('should set session active to false when signing out', async () => {
+    const response = await chai
+      .request(app)
+      .get('/api/v1/sessions/destroy')
+      .set('authorization', sessionToken);
+    expect(response).to.have.status(200);
+  });
+
+  it('respond 500 error if if there is an error', async () => {
+    const stubfunc = { destroy };
+    const sandbox = sinon.createSandbox();
+    sandbox.stub(stubfunc, 'destroy').rejects(new Error('Server Error'));
+
+    const next = sinon.spy();
+    const res = {
+      status: () => ({
+        json: next
+      })
+    };
+    await destroy({}, res);
+    sinon.assert.calledOnce(next);
```

Comment:
why is signout calling `next`

Reply:
I called next to confirm that the spy function was called once.